packages/cactus-plugin-ledger-connector-besu/src/test/typescript/integration/plugin-ledger-connector-besu/deploy-contract/testLedgerParameters.ts

```diff
@@ -0,0 +1,76 @@
+// tslint:disable-next-line: no-var-requires
+const tap = require("tap");
+import { BesuTestLedger } from "@hyperledger/cactus-test-tooling";
+
+tap.test("deploys a Besu node with the default configurations", async (assert: any) => {
+    assert.plan(2);
+
+    // No options
+    const besuTestLedger = new BesuTestLedger();
+    await besuTestLedger.start();
+
+    assert.tearDown(async () => {
+        await besuTestLedger.stop();
+        await besuTestLedger.destroy();
+    });
+
+    assert.ok(besuTestLedger);
+    assert.end();
+});
+
+tap.test("deploys a Besu Node on the Rinkeby network", async (assert: any) => {
+    assert.plan(2);
+    let RinkebyNetworkParameters = "--mount type=bind,source=/<myvolume/besu/testnode>,target=/var/lib/besu hyperledger/besu:latest --miner-enabled --miner-coinbase fe3b557e8fb62b89f4916b721be55ceb828dbd73--network=dev --min-gas-price=0";
```

Comment:
Not sure if the linter catches this at present or not, but variables should be camel-case, e.g. starting with lowercase.
I'll double check if the linter complains and if not, will make sure that it does so in the future.

Reply:
It doesn't complain about that, maybe that's why it went unnoticed.

Reply:
> It doesn't complain about that, maybe that's why it went unnoticed.

That's fair. It isn't documented anywhere that it should be camel case so if the linter let's it through then it is to be assumed that everything is as per the style guide (which we don't have explicitly, just through the linter configuration, basically the linter configuration is the style guide)
I'll check on the config.